components/Checkbox.vue

```diff
@@ -273,11 +280,15 @@
 		@include circle;
 		animation: pressing-back $duration-half $ease-in alternate 2;
 
-		:comp:focus & {
+		:comp:any-hover &,
+		:comp:focus-visible & {
 			@include large-shadow-unchecked-focus;
 		}
 
-		:comp.checked:focus & {
+		:comp:any-hover input:checked + &,
+		:comp:any-hover input:indeterminate + &,
+		:comp:focus-visible input:checked + &,
+		:comp:focus-visible input:indeterminate + & {
```

Comment:
```suggestion
		:comp:is(:any-hover, :focus-visible) input:is(:checked, :indeterminate) + & {
```
注意，请务必测试，有小概率可能不能用，但绝对比现在的写法优雅许多。